docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,147 @@
+# Tutorial: Deploy the Chrony charm
+
+## What you'll do
+This tutorial will walk you through deploying the chrony charm; you will:
```

Comment:
Nitpick: Capitalise Chrony (assuming it needs to be capitalised; it seems like it based on the rest of the tutorial)

Reply:
Hmm, maybe we should standardize all mentions to use the lowercase "chrony"? I did some research online, it appears that the official documents and articles mostly use the lowercase version.

Some examples:
Official chrony website: [chrony-project.org](https://chrony-project.org)
Chrony FAQ: [chrony-project.org/faq.html](https://chrony-project.org/faq.html)
Article by the current chrony maintainer: https://www.redhat.com/en/blog/combining-ptp-ntp-get-best-both-worlds

Reply:
Thanks for doing the research! I'm fine with using lowercase "chrony", as long as it's consistent throughout the documentation :)

Reply:
Thanks, replaced all instances of "Chrony" with "chrony"